app/src/main/java/org/dhis2/data/forms/dataentry/fields/LayoutProviderImpl.kt

```diff
@@ -14,7 +13,6 @@ import org.hisp.dhis.android.core.common.ValueType
 import org.hisp.dhis.android.core.common.ValueTypeRenderingType
 
 private val layouts = mapOf<KClass<*>, Int>(
-    DisplayViewModel::class to R.layout.custom_form_display,
```

Comment:
shouldn't you add R.layout.custom_form_display in the methods below to provide it?

Reply:
There is no need because this viewModel was used to display indicators in the form. That feature was removed and the indicators were move to the analytics module, which has it's own indicators model.